cargo-miri/bin.rs

```diff
@@ -586,11 +587,19 @@ fn phase_cargo_miri(mut args: env::Args) {
     };
     let subcommand = match &*subcommand {
         "setup" => MiriCommand::Setup,
-        "test" | "t" | "run" | "r" => MiriCommand::Forward(subcommand),
+        "test" | "t" | "run" | "r" => MiriCommand::Forward(vec![subcommand]),
+        "nextest" => {
+            // nextest requires a subcommand (e.g. nextest run) to be specified before --target, so
```

Comment:
Oh, it's the forced `--target`.
I think we should be able to find something better here than having nextest-specific logic in cargo-miri. We are already adding `--target-dir` after forwarding the user-provided options, the same should work for `--target`?

Reply:
https://github.com/rust-lang/miri/pull/2402 should do the reordering.
Then I hope this PR can be just adding "nextest" to the match.